### DISCUSSION TITLE
How to output raw GPS data to a UART?

### DISCUSSION THREAD

**Discussion by clodzeischi** [iNavFlight/inav#10144] [Q&A]
Need to connect Raspberry Pi running Kismet to an FC and pull GPS data into GPSD socket. Is there a way to do it directly using NMEA or UBLOX without running it through an MSP translator?

**Comment by sensei-hacker** [comment]
You can vampire tap the wire from the GPS tx to the FC rx.
I made a tracker which works this way. This UBlox passes right through the tracker.
